src/Module.php

```diff
@@ -39,4 +44,10 @@ public function init($moduleManager)
             'getFilterConfig'
         );
     }
+
+    /** @return ServiceManagerConfiguration */
+    public function getFilterConfig(): array
+    {
+        return (new ConfigProvider())()['filters'];
```

Comment:
I'd rather provide a public method in `ConfigProvider` than fetching the array key but thats most probably just my personal preference.

Reply:
> I'd rather provide a public method in `ConfigProvider`…

This makes sense, because it is also used elsewhere:

```php
public function getPluginManager(): FilterPluginManager
{
    $plugins = $this->plugins;
    if (! $plugins instanceof FilterPluginManager) {
        $plugins = new FilterPluginManager(new ServiceManager(), (new ConfigProvider())()['filters']);
        $this->setPluginManager($plugins);
    }

    return $plugins;
}
```

https://github.com/laminas/laminas-filter/pull/136/files#diff-3a6c9663e22963eb296b993e0b7cc8335c9f7f02f8364fc66385ed7175b5639dR124

Reply:
Yes - referencing array keys is like a hidden public API and worse than simply exposing a public method 👍

Reply:
I've removed filter config completely from the config provider in favour of a class constant on the plugin manager, as per `laminas-serializer` thanks to @boesing